.gitignore

```diff
@@ -92,9 +92,6 @@ node_modules/
 # Yarn Integrity file
 .yarn-integrity
 
-# Yarn Lock file
-yarn.lock
```

Comment:
We should replace this with `package-lock.json`

Reply:
Nice one, I've added it.